### ISSUE TITLE
[DOCUMENTS]: Documents MUST be accessible, Covid Links and Resources PDF

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8172]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
! This is really a 508-defect-1, but since we need to coordinate with another 
! team to find out who is responsible for creating the accessible PDF, I am 
! setting this as a 2.
```

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Documents **must** be accessible. The [Covid-19 Links and Resources PDF](https://www.va.gov/covid-19-links-and-resources.pdf) fails an accessibility check, and must be remediated so that all the content is accessible for screen readers and other assistive technology.

If this PDF is created by another team, they may need to remediate the PDF. I am not clear on the responsibility assignment. Depending on what application is used to create the PDF, there are export options that will support accessibility.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to access all of the content of PDFs, just as sighted users may.

## Environment

* Operating System: all
* Browser: all
* Server destination: production

## Steps to Recreate

1. Access the PDF at https://www.va.gov/covid-19-links-and-resources.pdf
1. Open the PDF in Adobe Acrobat Pro
1. Navigate to the Tools tab, and scroll to the bottom
1. In the second to the last row, select Accessibility
1. Run a Full Check
1. Verify there are Failed items, such as Tagged PDF, Title, in the Document; additional issues in Page Content, etc.

## Possible Fixes (optional)

Using the Adobe Acrobat Pro Accessibility Checker, follow the steps listed in the first link below to evaluate and remediate the PDF. Frankly, the first few times can really try your patience! After a while, it becomes clearer, and this is a valuable skill to add to your resume. 

## WCAG or Vendor Guidance (optional)

* [Recommended: Using the Acrobat Pro DC Accessibility Checker](https://www.adobe.com/accessibility/products/acrobat/using-acrobat-pro-accessibility-checker.html)
* [HHS.gov - PDF File 508 Checklist (WCAG 2.0 Refresh)](https://www.hhs.gov/web/section-508/making-files-accessible/checklist/pdf/index.html)
* [Make Sure PDF Documents are Tagged for Accessibility, Deque University](https://dequeuniversity.com/tips/tagged-pdf-documents)
* [Requirements for an Accessible PDF: Part 1 (addl parts linked), Deque](https://www.deque.com/blog/requirements-accessible-pdf/)
* [PDF Accessibility: Everything You Need to Know, Deque](https://www.deque.com/blog/pdf-accessibility/)
* [W3C PDF Techniques for WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/pdf)
* [Adobe PDF Accessibility Overview](https://www.adobe.com/accessibility/pdf.html)

## Screenshots or Trace Logs

![Screen Shot 2020-04-20 at 10 57 59 AM](https://user-images.githubusercontent.com/57469/79766455-0115e800-82f6-11ea-8d4e-1929f8a5e94a.png)

**Comment by jenstrickland** [comment]
@stephorkandatx I think this can be closed, since Lauren worked with the stakeholder to get the PDFs accessible and I validated them. Any updates, I'll need to review.
